src/backend/api/reviews.js

```diff
@@ -0,0 +1,71 @@
+const express = require('express');
+const { sum } = require('../database');
+const router = express.Router();
+const knex = require('../database');
+
+// Returns all reviews
+// GET api/reviews/
+router.get('/', async(request, response) => {
+    try {
+        const getAllReviews = await knex('review'); //.select('title');
+        response.json(getAllReviews);
+    } catch (error) {
+        throw error;
+    }
+});
+
+// Adds a new review
+// POST api/reviews/
+router.post('/', async(request, response) => {
+    try {
+        const postReservation = await knex('review').insert({
+            title: request.body.title,
+            description: request.body.description,
+            stars: request.body.stars,
+            created_date: request.body.created_date,
+            meal_id: request.body.meal_id,
+        });
+        response.json(postReservation);
+    } catch (error) {
+        throw error;
```

Comment:
Catching only to re-throw isn't very useful. You could just as well remove the try/catch.
Or you could log the error message so you can see in your logs what went wrong but it would also be good to perhaps send a error response to the user with a 500 HTTP code.

Reply:
i added these lines. is this like this ?